loopchain/baseservice/rest_client.py

```diff
@@ -154,7 +154,9 @@ def _call_jsonrpc(self, target: str, method: RestMethod, params: Optional[NamedT
         url = self._create_jsonrpc_url(target, method)
         http_client = HTTPClient(url)
         request = self._create_jsonrpc_params(method, params)
-        return http_client.send(request, timeout=timeout)
+
+        response: Response = http_client.send(request, timeout=timeout)
+        return response.data.result
```

Comment:
`response.data` is `Union[JSONRPCResponse, List[JSONRPCResponse]]` type and has `result` instance variable in SuccessResponse.
It would be better to process when single response and assert `response.data` is SuccessResponse.

Reply:
how about resolve risks of batch responses by:

```py
        response: Response = http_client.send(request, timeout=timeout)
        if isinstance(response.data, list):
            raise NotImplementedError(f"Received batch response. Data: {response.data}")
        else:
            return response.data.result
```